src/gui/LoginLogic.js

```diff
@@ -16,7 +16,7 @@ async function sendLoginRequest() {
         else {
             document.getElementById("loginFail").hidden = true;
             document.getElementById("login").hidden = true;
-            document.getElementById("AdminView").hidden = false;
+            document.getElementById("studentView").hidden = false;
```

Comment:
I modified this in my branch as well to support all three roles, but I will resolve the conflicts when it comes to it.